Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -33,20 +35,66 @@ public GameObject ThumbRoot
             }
         }
 
-        [Range(0, 1)]
+        #region MyValue 
+        [SerializeField]
+        private int minValue = -4;
+        [SerializeField]
+        private int maxValue = 4;
+
+        [SerializeField]
+        private bool wholeNumbers = true;
+
         [SerializeField]
+        private float startValue = 0f;
+
+        [Tooltip("Automatic placement of slider's steps.")]
+        [SerializeField]
+        private bool tickMarksAutoFill = true;
```

Comment:
Also nit: It would be awesome to expand this comment more to explain what it's doing - i.e. how is it doing automatic placement?